src/ilasm/writer.cpp

```diff
@@ -416,7 +416,11 @@ HRESULT Assembler::CreateExportDirectory()
     memset(&exportDirIDD,0,sizeof(IMAGE_EXPORT_DIRECTORY));
     // Grab the timestamp of the PE file.
     DWORD fileTimeStamp;
-    if (FAILED(hr = m_pCeeFileGen->GetFileTimeStamp(m_pCeeFile,&fileTimeStamp))) return hr;
+    if (FAILED(hr = m_pCeeFileGen->GetFileTimeStamp(m_pCeeFile,&fileTimeStamp)))
+    {
+        delete [] pAlias;
```

Comment:
Please use the holder pattern instead of putting delete at many places. All you need to do is to include `holder.h` and declare the pAlias as
```C++
NewArrayHolder<char*> pAlias;
```
The deletion will automatically happen when the holder goes out of scope.

Reply:
@janvorli Thanks! Adjusted according to your suggestion.